jsbi.mjs

```diff
@@ -1783,4 +1847,14 @@ JSBI.__kBitConversionBuffer = new ArrayBuffer(8);
 JSBI.__kBitConversionDouble = new Float64Array(JSBI.__kBitConversionBuffer);
 JSBI.__kBitConversionInts = new Int32Array(JSBI.__kBitConversionBuffer);
 
+// DataView polyfills
+JSBI.DataViewSetBigUint64 =
+  JSBI.__makeDataViewSetter('DataViewSetBigUint64', 'Uint32');
+JSBI.DataViewSetBigInt64 =
+  JSBI.__makeDataViewSetter('DataViewSetBigInt64', 'Int32');
+JSBI.DataViewGetBigUint64 =
+  JSBI.__makeDataViewGetter('DataViewGetBigInt64', 'Uint32');
```

Comment:
`DataViewGetBigInt64` or `DataViewGetBigUInt64`?

Reply:
Good catch, it's a typo in the function name.